shared/data/constants.js

```diff
@@ -8,7 +8,8 @@ let constants = {
     "entityMap": "data/tracker_lists/entityMap.json",
     "easylists": ["privacy", "general"],
     "blocking": ["Advertising", "Analytics"],
-    "httpsWhitelist": "data/httpsWhitelist.json",
+    "httpsUpgradeList": "data/httpsUpgradeList.json",
+    "httpsUpgradeListUrl": "https://brian.duckduckgo.com/contentblocking.js?l=https2",
```

Comment:
This part isn't mergeable for now. Demarcating PR as WIP.

Reply:
We should discuss how to handle the backend piece. This PR won't work with the existing endpoint unless we modify it to do so.

The existing endpoint uses a JSON structure that makes working with the data more difficult than it needs to be and unless there's any reason not to, I'd like to change that along with deploying the larger list.

My preference would be to get the latest list from Zach, merge it with this PR and deploy it ahead of deploying this extension change: https://dub.duckduckgo.com/duckduckgo/ddg/pull/11741

Reply:
Yup. Just saying so it's not merged by anyone yet.